app/components/autocomplete/autocomplete.tsx

```diff
@@ -124,10 +126,10 @@ const Autocomplete = ({
             s.push(style.shadow);
         }
         if (isSearch) {
-            s.push(style.base, style.searchContainer, {height: maxListHeight});
+            s.push(style.base, paddingTop ? {top: paddingTop} : style.searchContainer, {height: hasElements ? maxListHeight : 0});
```

Comment:
The style probably has been brought from V1. Wonder if it makes sense removing it (not sure if there is any case were paddingTop is 0 or undefined, or at least rethinking it to use some proper constants, instead of magic numbers.

Reply:
Once I placed the `Autocomplete` component correctly inside `Search` this worked as expected without any changes.